### PR TITLE
Finalize Dataflow Read Pub/Sub Write BigQuery sample

### PR DESCRIPTION
**Pull request by damondouglas**
This PR finalizes the Dataflow Read Pub/Sub Write BigQuery sample by adding screenshots to guide the developer how to use the sample as well as some additional patch edits:
- Enable BigQuery service via Infrastructure-as-Code
- Directions how to get started in IntelliJ
- A call to fill out a survey

### CODE REVIEW

File: java/java-dataflow-samples/read-pubsub-write-bigquery/README.md [hunk 1]
```diff
@@ -66,8 +66,109 @@ Start your journey at the main method here:
...
+The previous provision required resources step will generate this for you and
+download to your machine.  Move this downloaded `gradle.properties` file to your
+IDE working directory on your local machine.
+
+If you lost it, you can download it again from Google Cloud Storage:
+1. Find the Google Cloud storage bucket containing the `gradle.properties` file.
+Cloud Code's Google Cloud Storage allows you to list and view buckets and
+objects.  Just locate the `gradle.properties` file and copy the bucket name.
+![copy_bucket_name_to_clipboard](img/copy_bucket_name_to_clipboard.png)
+2. [Open the IntelliJ terminal](https://www.jetbrains.com/help/idea/terminal-emulator.html)
```
Comment (ivanporty) [inline java/java-dataflow-samples/read-pubsub-write-bigquery/README.md]: so at this point we are talking "if you lost it", and we should direct users to use Cloud Code Terminal where Cloud SDK is preconfigured - Tools -> Google Cloud Code -> Tools -> Cloud Code Terminal

File: java/java-dataflow-samples/read-pubsub-write-bigquery/README.md [hunk 2]
```diff
@@ -66,8 +66,109 @@ Start your journey at the main method here:
...
+and run the following command.  NOTE: `<bucket>` is the name of the bucket
+you copied in the previous step.
+```sh
+gsutil cp gs://<bucket>/gradle.properties .
+```
+
+NOTE: If you would like to generate yourself: See template: [java/java-dataflow-samples/read-pubsub-write-bigquery/infrastructure/03.io
+/gradle.properties.tmpl](infrastructure/03.io/gradle.properties.tmpl).
+
+## Select the included IntelliJ Run Configuration
```
Comment (ivanporty) [inline java/java-dataflow-samples/read-pubsub-write-bigquery/README.md]: should we mention a user should be logged in before running or an error message will state that clearly? We can say something like "Use Tools -> Google Cloud Code -> Sign in to Google Cloud Platform..."

File: java/java-dataflow-samples/read-pubsub-write-bigquery/README.md (hunk 1 above)
Comment (damondouglas) [reply java/java-dataflow-samples/read-pubsub-write-bigquery/README.md]: Ah good point

File: java/java-dataflow-samples/read-pubsub-write-bigquery/README.md (hunk 2 above)
Comment (damondouglas) [reply java/java-dataflow-samples/read-pubsub-write-bigquery/README.md]: I'm putting Tools -> Google Cloud Code -> Sign in to Google Cloud Platform as the first requirement instead as well as repeated  under the Getting Started with IntelliJ section.

**Review by ivanporty — APPROVED** [review]
LGTM, thank you Damon!

**Review by averikitsch — APPROVED** [review]

**Pull request merged**